sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -440,6 +445,9 @@ public partial class BlobDownloadDetails
         public string EncryptionKeySha256 { get { throw null; } }
         public string EncryptionScope { get { throw null; } }
         public Azure.ETag ETag { get { throw null; } }
+        public bool HasLegalHold { get { throw null; } }
+        public System.DateTimeOffset? ImmutabilityPolicyExpiresOn { get { throw null; } }
+        public Azure.Storage.Blobs.Models.BlobImmutabilityPolicyMode? ImmutabilityPolicyMode { get { throw null; } }
```

Comment:
should we use `BlobImmutabilityPolicy` and one property for this?

Reply:
We did not group any other properties in `BlobProperties` in the past.  For example, the copy-related and access tier-related properties are all flat.

Reply:
Also, we can let One SDK decide in design review on Monday.